doc/newfeat.texi

```diff
@@ -4,117 +4,75 @@
 Changes and New Features in development version:
 @itemize @bullet
 
-@item @ESS{[R]}: The @code{in} keyword is now only fontified when
-inside a @code{for} construct. This avoids spurious fontification,
-especially in the output buffer where `in` is a commond English word.
-
-@item @ESS{[R]}: Fixed fontification toggling of certain syntactic
-elements such as @code{%op%} operators and backquoted function
-definitions.
-
-@item @ESS{[R]}: Fixed a bug that prevented @code{ess-font-lock-toggle-keyword}
-to be called interactively. This command asks with completion for a
-font-lock group to toggle. This functionality is equivalent to the
-font-lock menu.
-
-@item @ESS: Font-lock keywords are now generated lazily. That means you
-can now add or remove keywords from variables like @code{ess-R-keywords}
-in your Emacs configuration file after loading ESS (i.e. in the @code{:config}
-section for @code{use-package} users).
-
-@item @ESS{[R]}: @code{ess-r-package-use-dir} (which sets the working
-directory to the root of the current package) now works with any mode
-(e.g. in C or C++ files within @code{/src}).
-
-@item This is the last release to support Emacs older than 25.1
+@item This is the last release to support Emacs older than 25.1.
 Going forward, only GNU Emacs 25.1 and newer will be supported. Soon
 after this release, support for older Emacs versions will be dropped
 from the git master branch. Note that MELPA uses the git master branch
 to produce ESS snapshots, so if you are using Emacs < 25.1 from MELPA
 and are unable to upgrade, you should switch to MELPA-stable.
 
+@item ESS now displays the language dialect in the mode-line.
+So, for example, R buffers will now show ESS[R] rather than ESS[S].
+
+@item The ESS manual has been updated and revised.
+
+@item The ESS initialization process has been further streamlined.
+If you update the autoloads (which installation from
+@code{package-install} does), you should not need to @code{(require
+'ess-site)} at all, as autoloads should automatically load ESS when it
+is needed (e.g. the first time an R buffer is opened). In order to
+defer loading your ESS config, you may want to do something like
+@code{(with-require-after-load "ess" <ess-config-here>)} in your Emacs
+init file. Users of the popular @code{use-package} Emacs package can
+now do @code{(use-package ess :defer t)} to take advantage of this
+behavior.
+
+@item ESS now respects Emacs conventions for keybindings.
+This means that The @code{C-c [letter]} bindings have been
+removed. This affects @code{C-c h}, which was bound to
+@code{ess-eval-line-and-step-invisibly} in @code{sas-mode-local-map};
+@code{C-c f}, which was bound to @code{ess-insert-function-outline} in
+@code{ess-add-MM-keys}; and @code{C-c h}, which was bound to
+@code{ess-handy-commands} in @code{Rd-mode-map},
+@code{ess-noweb-minor-mode-map}, and @code{ess-help-mode-map}
+
+@item @ESS{[R]}: @code{ess-r-package-use-dir} now works with any mode.
```

Comment:
I don't understand this entry. Can we make it more user friendly without mentioning `ess-r-package-use-dir` which is an internal function?

Reply:
`ess-r-package-use-dir` isn't internal. It's interactive. I don't think this entry is confusing; what do you not like about it?

Reply:
Nevermind. I didn't know we have this function. Thought it's something else.

Reply:
OK, merged